libs/datasets/timeseries.py

```diff
@@ -1031,10 +1031,84 @@ def drop_regions_without_population(
 LOCATION_ID_AGG = "location_id_agg"
 
 
+@dataclass(frozen=True)
+class AverageStaticWeightAggregation:
```

Comment:
Would `WeightedAverageAggregation` be a better name? I'm not sure what role "static" is playing in the name

Reply:
The weights are static, instead of time-varying.